aries-site/src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,111 @@
+import { BestPracticeGroup, Example } from '../../layouts';
+import {
+  AvatarAccessibilityExample,
+  AvatarCardExampleDo,
+  AvatarCardExampleDont,
+  AvatarDataTableExampleDo,
+  AvatarDataTableExampleDont,
+  AvatarPageHeaderExample,
+  AvatarSizePreviewExample,
+} from '../../examples';
+
+## Anatomy
+
+Avatars can be displayed as an image, initials, or a logo. They come in three different sizes:
+
+- `small`: 24x24px
+- `medium`: 48x48px
+- `large`: 96x96px
+
+The container of an Avatar can be circular or a rounded square based on the use case.
+
+- Use a circle Avatar when representing a person either through an image or their initials.
+- Use a rounded square Avatar when representing a company’s or entity’s logo. This can be achieved by passing `round='medium'`
+- Always round the edges of a square container when using an avatar.
+
+<Example
+  figma="https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/avatar/AvatarSizePreviewExample.js"
+>
+  <AvatarSizePreviewExample />
+</Example>
+
+## Accessibility
+
+Inform the user of what the Avatar is and what will happen when it is clicked by including an `a11yTitle`.
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/avatar/AvatarAccessibilityExample.js"
+  width={{ min: 'medium' }}
+>
+  <AvatarAccessibilityExample />
```

Comment:
Can we include a clickable avatar in this example too since we reference it in the guidance?

Reply:
Good suggestion

Reply:
Yes I can add

Reply:
done